scripts/tests/yaml/runner.py

```diff
@@ -304,7 +304,7 @@ def run(parser_group: ParserGroup, adapter: str, stop_on_error: bool, stop_on_wa
 @test_runner_options
 @websocket_runner_options
 @pass_parser_group
-def websocket(parser_group: ParserGroup, adapter: str, stop_on_error: bool, stop_on_warning: bool, stop_at_number: int, show_adapter_logs: bool, show_adapter_logs_on_error: bool, use_test_harness_log_format: bool, delay_in_ms: int, server_address: str, server_port: int, server_path: str, server_name: str, server_arguments: str):
+def websocket(parser_group: ParserGroup, adapter: str, stop_on_error: bool, stop_on_warning: bool, stop_at_number: int, show_adapter_logs: bool, show_adapter_logs_on_error: bool, use_test_harness_log_format: bool, delay_in_ms: int, server_address: str, server_port: int, server_path: str, server_name: str, server_arguments: str, endpoint: int):
```

Comment:
Does the endpoint variable get used anywhere in here?

Reply:
I think it's needed here because of `click` and the way it handles the sets of arguments. 

I think because this function is decorated with the `@test_runner_options` it needs it. 

So, that probably means I need it everywhere that has that decorator. 

If I am honest, I only added it here because it doesn't run if I don't add it here :upside_down_face: 

Guess I need to RTFM on click / summon Vivien.